adr/0007-choosing-an-open-source-component.md

```diff
@@ -0,0 +1,57 @@
+# Choosing an open source component
+
+Date: 2018-01-09
+
+## Status
+
+Proposed.
```

Comment:
I'd tag as many people as possible here, since this has wide-ranging implications